src/native-common/AccessibilityUtil.ts

```diff
@@ -53,7 +60,38 @@ const componentTypeMap = {
     [Types.AccessibilityTrait.Radio_button_unchecked]: 'radiobutton_unchecked'
 };
 
+// Platform specific helpers exposed through Native-Common AccessibilityUtil. 
+export abstract class NativeHelpers {
```

Comment:
Consider giving a more specific name to this class to be derived by each platform. Maybe someone like CommonNativeAccessibilityUtil which would be in line with the pattern used currently or NativeAccessibilityHelpers. NativeHelpers is very generic and makes it's difficult to the reader to understand what it's about on its own.